models/articles.models.js

```diff
@@ -55,4 +55,15 @@ function checkArticleExists(article_id) {
         })
 }
 
-module.exports = { fetchArticleById, fetchAllArticles, fetchAllCommentsByArticleId, checkArticleExists }
+function updateArticleVotes(article_id, inc_votes) {
+    return db.query(
+        `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *`, [inc_votes, article_id])
+        .then(({rows}) => {
+            if (rows.length === 0) {
```

Comment:
If you're checking that the article exists before this check, will this ever reject?